features/recipes/router.py

```diff
@@ -115,11 +118,81 @@ def create_recipe(create_recipe_input_model: CreateRecipeInputModel):
     :return:
     """
     try:
-        created_recipe = features.recipes.operations.create_recipe(**create_recipe_input_model.model_dump())
+        created_recipe = features.recipes.operations.create_recipe(**create_recipe_input_model.__dict__)
 
         return features.recipes.responses.Recipe(**created_recipe.__dict__)
     except features.recipes.exceptions.CategoryNotFoundException:
         raise fastapi.HTTPException(
             status_code=fastapi.status.HTTP_400_BAD_REQUEST,
             detail=f"Category with id {create_recipe_input_model.category_id} does not exist"
-        )
\ No newline at end of file
+        )
+
+
+@recipes_router.post('/{recipe_id}/instructions', status_code=fastapi.status.HTTP_201_CREATED)
+def create_instructions(instructions_request: list[InstructionInput], recipe_id: int):
```

Comment:
We don't need that anymore, instructions can be crated during the recipe creation and updated after that

Reply:
We can have put request on /recipes/{recipe_id}/instructions and update the whole list of instructions at once